plugin/trino-pulsar/src/main/java/io/trino/plugin/pulsar/PulsarColumnMetadata.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pulsar;
+
+import io.trino.spi.connector.ColumnMetadata;
+import io.trino.spi.type.Type;
+
+import java.util.Objects;
+
+/**
+ * Description of the Trino Pulsar column metadata.
+ */
+public class PulsarColumnMetadata
+        extends ColumnMetadata
+{
+    public static final String KEY_SCHEMA_COLUMN_PREFIX = "__key.";
+
+    private boolean isInternal;
+    // need this because trino ColumnMetadata saves name in lowercase
+    private String nameWithCase;
+    private PulsarColumnHandle.HandleKeyValueType handleKeyValueType;
+
+    private DecoderExtraInfo decoderExtraInfo;
+
+    public PulsarColumnMetadata(String name, Type type, String comment, String extraInfo,
```

Comment:
We usually place each argument in separate line. Please follow that here and everywhere else.

Reply:
updated.